src/lib.rs

```diff
@@ -179,9 +184,14 @@ impl AuraedRuntime {
         //runtime::hydrate(&db).await?;
 
         // Event loop
-        let res = handle.await.unwrap();
+        let res = handle.await?;
+        match res {
+            Ok(_) => {
+                info!("{:?}", res);
```

Comment:
`res` is currently a `Result<(), Error>` so the `Ok` variant doesn't seem useful to print out (maybe it will change and that's why it was being printed out before too?).